bdist_rpm-no-gpu.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+set -x
+
+SPEC=slurm-no-gpu.spec
+
+VERSION=`grep "Version:.*[0-9]" $SPEC | tr -s " " |  awk '{print $2;}'`
+RELEASE=`grep "%global rel.*[-1-9]" $SPEC | tr -s " " | awk '{print $3}'`
+
+if [ ${RELEASE} != "1" ]; then
+    SUFFIX=${VERSION}-${RELEASE}
+else
+    SUFFIX=${VERSION}
+fi
+
+GITTAG=$(git log --format=%ct.%h -1)
+
+mkdir -p BUILD SOURCES SPECS RPMS BUILDROOT
+git archive --format=tar.gz -o "SOURCES/slurm-${SUFFIX}.tar.gz" --prefix="slurm-${SUFFIX}/" HEAD
+cp $SPEC "SPECS"
+
+# there's no option to pass nvml, it is only autodetected
+# nvidia-driver-devel provides the libnividia-ml.so symlnk, the real .so.1 comes from nvidia-driver-NVML
+# cuda-nvml-dev provides the nvml.h (put is part of the cuda rpms so has cud version in the name, hence the *)
+# however, it's not in the default include paths nor the slurm hardcoded ones, so use CPATH
+# nvml api is not really cuda specific, last API is from cuda 5, so any recent cuda will do
+
+# TODO: what if more than one cuda is available/installed, then the * thingies will probably not work
+# pmix-3 as rebuild from github src.rpm includes the devel rpms in the rpm
+sudo yum remove -y nvidia-driver-devel cuda-nvml-dev-10-2 nvidia-driver-NVML 
+sudo yum install -y ucx-devel "pmix-devel > 3.0.0" numactl-devel hwloc-devel
+
+# remove json-c12 -> use plain json-c
```

Comment:
unless there is some hackery going on here, this is not ok for el7 i think

Reply:
Well, we won't be having it on EL7, or will we?

Reply:
we are not going to update  the el7 clusters?